cmd/evm/main.go

```diff
@@ -174,17 +176,23 @@ type VMEnv struct {
 	Gas   *big.Int
 	time  *big.Int
 	logs  []vm.StructLog
+
+	evm *vm.EVM
 }
 
-func NewEnv(state *state.StateDB, transactor common.Address, value *big.Int) *VMEnv {
-	return &VMEnv{
+func NewEnv(state *state.StateDB, transactor common.Address, value *big.Int, cfg *vm.Config) *VMEnv {
+	params.HomesteadBlock = new(big.Int)
```

Comment:
This is the EVM intepreter utility binary. I don't think that's going to matter. In any case the number is settable in my follow up PR.